capstan.go

```diff
@@ -152,6 +152,8 @@ func main() {
 				cli.StringFlag{Name: "boot", Usage: "specify config_set name to boot unikernel with"},
 				cli.BoolFlag{Name: "persist", Usage: "persist instance parameters (only relevant for qemu instances)"},
 				cli.StringSliceFlag{Name: "env", Value: new(cli.StringSlice), Usage: "specify value of environment variable e.g. PORT=8000 (repeatable)"},
+				cli.StringSliceFlag{Name: "volume", Value: new(cli.StringSlice), Usage: `{path}[:{key=val}], e.g. ./volume.img:format=raw (repeatable)
+				Default options are: format=raw, aio=native, cache=none`},
```

Comment:
Please, replace default options with the actual string, i.e. `format=raw:aio=native:cache=none` as this will help understand users how to pass the values (the format in the usage mentions `:` as a separator, but here you write default opts in a comma-separated list).